tests/schemas/api_v3/block.schema.js

```diff
@@ -14,17 +14,22 @@
  *
  */
 import Joi from 'joi';
+import regex from './regex';
+
+const generator = {
+	address: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
+};
 
 const blockSchema = {
-	id: Joi.string().required(),
+	id: Joi.string().pattern(regex.HASH_SHA256).required(),
 	height: Joi.number().integer().min(0).required(),
 	version: Joi.number().required(),
 	timestamp: Joi.number().integer().min(1).required(),
-	generatorAddress: Joi.string().allow('').required(),
-	assetsRoot: Joi.string().required(),
-	stateRoot: Joi.string().required(),
-	transactionRoot: Joi.string().required(),
-	previousBlockID: Joi.string().required(),
+	generator: Joi.object(generator).required(),
+	assetRoot: Joi.string().pattern(regex.HASH_SHA256).required(),
+	stateRoot: Joi.string().pattern(regex.HASH_SHA256).required(),
+	transactionRoot: Joi.string().pattern(regex.HASH_SHA256).required(),
+	previousBlockID: Joi.string().pattern(regex.HASH_SHA256).required(),
 	signature: Joi.string().allow('').required(),
 	aggregateCommit: Joi.object().required(),
```

Comment:
Is it possible to specify the object structure for `aggregateCommit` here? Kindly ensure the same is reflected in the gateway source mappings.